node/libs/roles/src/proto/validator.proto

```diff
@@ -11,7 +11,8 @@ message Fork {
 
 message Genesis {
   optional Fork fork = 1; // required
-  repeated PublicKey validators = 2;
+  repeated PublicKey validators = 2  [deprecated = true];
```

Comment:
document that validators and validators_v1 are mutually exclusive. Technically you should be allowed to put those fields into a oneof, but buf is too dumb to recognize this as a backward compatible change: https://buf.build/docs/breaking/rules#field_same_oneof